__tests__/components/__snapshots__/dashtable_v2.test.jsx.snap

```diff
@@ -86,11 +86,9 @@ exports[`GlobalDashboardTable renders dashtable with correct data format 1`] = `
           style="padding: 10px; border: 1px solid grey; text-align: center; width: 50%;"
         >
           <a
-            href="/dashboard/v2/details/cljbqrfoa0006kohk894ym6v3/student[A]@gmail.com"
+            href="/dashboard/v2/details?studentName=student%5BA%5D%40gmail.com&classroomName="
```

Comment:
I dug a bit deeper into this. I think what's happening is React router encodes when pushing the URL to the history. If you're unfamiliar with encoding/decoding of URL params, checkout [this article](https://stackdiary.com/tutorials/decoding-and-encoding-urls-in-javascript/).

I think what needs to be done is prevent the URL from being encoded by the React router, maybe there's a way to decode before the URL gets generated. Checkout the following posts for some hints. [Post 1 ](https://stackoverflow.com/questions/48523058/encoding-uri-using-link-of-react-router-dom-not-working)& [Post 2](https://github.com/remix-run/history/issues/505)